wg-charter.html

```diff
@@ -0,0 +1,406 @@
+        <p>Normative specification changes are generally expected to have a corresponding set of tests, either in the form of new tests or modifications to existing tests, or must include the rationale for why test updates are not required for the proposed update.</p>
+      </section>
+
+
+      <section id="coordination">
+        <h2>Coordination</h2>
+        <p>For all specifications, this Working Group will seek <a href="https://www.w3.org/Guide/Charter.html#horizontal-review">horizontal review</a> for accessibility, internationalization, performance, privacy, and security with the relevant Working and Interest Groups, and with the <a href="https://www.w3.org/2001/tag/" title="Technical Architecture Group">TAG</a>. Invitation for review must be issued during each major standards-track document transition, including <a href="https://www.w3.org/2018/Process-20180201/#RecsWD" title="First Public Working Draft">FPWD</a> and <a href="https://www.w3.org/2018/Process-20180201/#RecsCR" title="Candidate Recommendation">CR</a>, and should be issued when major changes occur in a specification.</p>
+
+        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/2018/Process-20180201/#WGCharter">W3C Process Document</a>:</p>
+
+        <div>
+          <h3 id="w3c-coordination">W3C Groups</h3>
+          <dl>
+            <dt><a href="https://www.w3.org/2001/tag/">Technical Architecture Group (TAG)</a></dt>
+        <dd>The WG may ask the Technical Architecture Group to review some of its specifications.</dd>
+
+        <dt><a href="https://www.w3.org/2019/webapps/">Web Applications Working Group</a></dt>
+        <dd>The Web Applications Working Group (WebApps WG) produces specifications that facilitate the development of client-side web applications.</dd>
+
+        <dt><a href="https://www.w3.org/2011/webappsec/">Web Application Security Working Group</a></dt>
+        <dd>This Working Group develops security and policy mechanisms to improve the security of Web Applications, and enable secure cross-site communication.</dd>
+
+        <dt><a href="https://www.w3.org/wasm/">Web Assembly Working Group</a></dt>
+        <dd>This Working Group develops a size- and load-time-efficient format and execution environment, allowing compilation to the web with consistent behavior across a variety of implementations.</dd>
+
+        <dt><a href="https://www.w3.org/community/webmachinelearning/">Machine Learning for the Web Community Group</a></dt>
+        <dd>This Community Group incubates a dedicated low-level Web API for machine learning inference.</dd>
+
+        <dt><a href="https://www.w3.org/community/wicg/">Web Platform Incubator Community Group</a></dt>
```

Comment:
Three comments here:
1. Given there is a GPU for the web CG, are there actual proposals/incubations in the WICG that this WG wants to consider adopting? I suppose it's possible, but just curious why this group would want to cast the net so wide...
2. This must have been copy/pasted from Web Performance WG, since it still says "Web Performance Working Group" in the prose here (whoops!)
3. There are specific implementation criteria outlined for adoption from the WICG. I don't think this group needs to be so prescriptive on this point. Rather than rely on "implemented and available in at least one browser..." suggest dropping in favor of adoption by unanimous approval of the chairs, or other stipulation not based necessarily on implementation status.

Reply:
I dropped the WICG. I suspect this was the result of a copy-and-paste from the Web Performance WG charter...

Reply:
:)